src/frontend/screens/Settings/ProjectConfig.tsx

```diff
@@ -192,4 +240,12 @@ const styles = StyleSheet.create({
     justifyContent: "center",
     alignItems: "center",
   },
+  LeaveBttn: {
```

Comment:
nit, but curious why the Pascal-case here?

Reply:
just a typo!